continuous-pipe.yml.erb

```diff
@@ -0,0 +1,192 @@
+variables:
+  MYSQL_DATABASE: <%= config.name %>
+  MYSQL_USER: <%= config.name %>
+
+defaults:
+  cluster: ${CLUSTER}
+  environment:
+    name: '"<%= config.name %>" ~ code_reference.branch'
+
+tasks:
+  web_image:
+    build:
+      environment:
+        GITHUB_TOKEN: ${GITHUB_TOKEN}
+
+    services:
+      web:
+        image: quay.io/organisation/<%= config.name %>
+
+  storage:
```

Comment:
We've been condensing the deployment tasks for a few projects. Should we do the same here (put everything under the `deployment` task)?